src/components/Tabs/ContactComponent.tsx

```diff
@@ -168,6 +180,27 @@ export default function ContactComponent() {
                   </Modal>
                 </Flex>
               </Flex>
+              {failedtoSend && (
+                <div className="mt-4 flex h-14 w-full items-center gap-2 rounded-sm bg-red-100 px-4 text-sm text-red-500">
+                  <AlertTriangleIcon className="h-5 w-5" />
+                  <p className="font-semibold">
+                    Sending failed. Please try again.
+                  </p>
+                </div>
+              )}
+
+              <Button
+                type="submit"
+                width="219px"
+                height="47px"
+                color="white"
+                bg={colors[`${valid}`]}
+                _hover={{ bg: colors[`${valid}`] }}
```

Comment:
I might be wrong but I feel like width, height, color, bg color, and hover can probably all be done with Tailwind?